common/src/main/spark-4.0/org/apache/comet/shims/ShimCometConf.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.comet.shims
+
+trait ShimCometConf {
+  protected val COMET_SCHEMA_EVOLUTION_ENABLED_DEFAULT = true
+  protected val COMET_ANSI_MODE_ENABLED_DEFAULT = true
```

Comment:
Do we really want to enable this by default when we know that we are not compatible yet?

Reply:
I suppose we have to do this otherwise all plans will currently fall back to Spark because ANSI mode is enabled by default in Spark 4.

We should really only fall back to Spark for expressions that are actually affected by ANSI support rather than fall back for everything.

Perhaps we could add a note to the documentation explaining that ANSI mode in Spark 4 is not fully compatible yet.

Reply:
I am thinking to use Spark-4.0 as a test bed for ANSI readiness. I will send another PR for enabling Spark tests and if I find any incompatibility, I will disable them per function.

Reply:
You need it to be enabled for Spark tests, isn't?

Reply:
Also see https://github.com/apache/datafusion-comet/pull/509 to allow us to fall back per function if ANSI is not supported yet and https://github.com/apache/datafusion-comet/issues/313 where we are tracking all ANSI work